docusaurus/website/docusaurus.config.ts

```diff
@@ -1,35 +1,105 @@
-import type { Config } from '@docusaurus/types';
 import type * as Preset from '@docusaurus/preset-classic';
-import {
-  customFields,
-  generalConfig,
-  plugins,
-  presetsDocs,
-  presetsTheme,
-  themeConfigNavbar,
-  themeConfigFooter,
-  themeConfigPrism,
-  themeConfigColorMode,
-} from './docusaurus.config.base';
+import type { Config } from '@docusaurus/types';
+import dotenv from 'dotenv';
+import remarkFigureCaption from 'gridsome-remark-figure-caption';
+import { resolve } from 'node:path';
+import { themes, type PrismTheme } from 'prism-react-renderer';
+import redirects from './websiteRedirects.json';
 
-const devPortalHome = 'https://grafana.com/developers';
-const [docsFooterLinks, ...otherFooterLinks] = themeConfigFooter.links;
 const gitHash = process.env.GITHUB_SHA || 'local';
+const envFile = process.env.DEV_PORTAL_ENV === 'dev' ? '.env.development' : '.env.production';
+
+dotenv.config({ path: `${__dirname}/${envFile}`, encoding: 'utf8' });
+
+const PORTAL_URL = `https://${process.env.DEV_PORTAL_HOST}`;
+const PORTAL_HOME_HREF = `${PORTAL_URL}/developers`;
+
+// Replace background and color to better match Grafana theme.
+const grafanaPrismTheme: PrismTheme = {
+  ...themes.oneDark,
+  plain: {
+    color: 'rgb(204, 204, 220)',
+    backgroundColor: '#181b1f',
+  },
+};
 
 const config: Config = {
-  ...generalConfig,
-  url: 'https://grafana.com/',
+  title: 'Grafana Plugin Tools',
+  tagline: 'Scaffold a Grafana plugin with one command',
+  url: PORTAL_URL,
   baseUrl: 'developers/plugin-tools/',
-  plugins,
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.png',
+  // GitHub pages deployment config.
+  // If you aren't using GitHub pages, you don't need these.
+  organizationName: 'grafana', // Usually your GitHub org/user name.
+  projectName: 'plugin-tools', // Usually your repo name.
```

Comment:
@tolzhabayev we don't need this anymore, right? 🤔

Reply:
Correct, I think it can be removed